src/frontend/src/app/update-modal/update-modal.component.ts

```diff
@@ -168,9 +161,12 @@ export class UpdateModalComponent implements OnInit, OnChanges {
         if (data.error) {
           console.log('error in response', data.error);
           this.alertService.error(data.error.message);
+        } else {
+          this.alertService.success('Record updated successfully.');
```

Comment:
REcord? you mean `Schedule` ? Does not we get a informational message from REST api which we can display to user?

Reply:
There is no informational message from api side. We are only getting updated schedule record back in response.